### DISCUSSION TITLE
Custom Piano Music: Jingle Bells

### DISCUSSION THREAD

**Discussion by KarmaCreations** [Monika-After-Story/MonikaModDev#7792] [general - piano notes]
Since Monika mentioned that her favorite Christmas Carol is Jingle Bells, I decided to make it for the piano. Enjoy! (#3765)
Note: Due to a bug discussed in #5100, the first time it is played, you have to wait at the end of the first line until Monika recognizes that you haven't played anything before you can continue the piece.
[jinglebells.txt](https://github.com/Monika-After-Story/MonikaModDev/files/3958178/jinglebells.txt)
(As always, convert to .json before putting it into the folder.)
Here are the notes:
E 9 U Y E
E E E 9 U Y R
R O 9 U 6
P P O Y 9
E 9 U Y E
E 9 U Y R
R R O 9 U P P P
P [ P 9 U Y
P 9 9 9 9 9 9
9 P Y U 9
O O O O O 9 9
9 9 9 U U 9 U P
9 9 9 9 9 9
9 P Y U 9
O O O O O 9 9
9 9 P P O U Y

**Comment by Konya-chan** [comment]
Hello, man. Just passing to say just one single note is wrong.
Here, it's write "P P O Y 9", but in the code is "P P O U 9".
I'm curious about which one is the right lol

**Reply by Cini01** [reply]
From the sound I would say "PPOU9"? But before I read that "PPOY9" sounded right too?! xD

Btw, I think that "PPUY9" sounds a bit off and replaced it with "PPUYU" (I personally think it sounds right)
